IBPSA/Fluid/HeatExchangers/BaseClasses/PartialEffectivenessNTU.mo

```diff
@@ -253,6 +253,12 @@ for <code>UA</code>.
 </html>", revisions="<html>
 <ul>
 <li>
+February 3, 2023, by Jianjun Hu:<br/>
+Added <code>noEvent()</code> in the assertion function to avoid optimica error.<br/>
```

Comment:
Better write "to avoid Optimica to not converge" as an error can be many different things. Same for the message below in the other file.